sample/src/main/res/layout/activity_wheel.xml

```diff
@@ -0,0 +1,117 @@
+        android:layout_width="match_parent"
+        android:layout_gravity="center"
+        android:layout_below="@+id/desc_textView"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:android="http://schemas.android.com/apk/res/android">
+
+        <FrameLayout
+            android:id="@+id/mcLayout"
+            android:layout_width="wrap_content"
+            android:layout_gravity="center"
+            android:layout_marginLeft="@dimen/spin_margin"
+            android:layout_height="wrap_content">
+
+            <ImageView
+                android:id="@+id/mc_bg"
+                android:layout_width="wrap_content"
+                android:layout_gravity="center"
+                android:src="@drawable/spin_wheel"
+                android:layout_height="wrap_content" />
+
+            <com.szugyi.circlemenu.view.WheelLayout
+                android:id="@+id/mc_wheel"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center"
+                app:speed="50"
+                app:wheel_radius_ratio="@dimen/spin_radius_ratio"
+                app:firstChildPosition="East">
+                <include layout="@layout/wheel_items" />
+            </com.szugyi.circlemenu.view.WheelLayout>
+        </FrameLayout>
+
+        <FrameLayout
+            android:id="@+id/scLayout"
+            android:layout_width="wrap_content"
+            android:layout_gravity="center"
+            android:layout_marginRight="@dimen/spin_margin"
+            android:layout_height="wrap_content">
+
+            <ImageView
+                android:id="@+id/sc_bg"
+                android:layout_width="wrap_content"
+                android:layout_gravity="center"
+                android:src="@drawable/spin_wheel"
+                android:layout_height="wrap_content" />
+
+            <com.szugyi.circlemenu.view.WheelLayout
+                android:id="@+id/sc_wheel"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center"
+                app:speed="75"
+                app:wheel_radius_ratio="@dimen/spin_radius_ratio"
+                app:firstChildPosition="West">
+                <include layout="@layout/wheel_items" />
+            </com.szugyi.circlemenu.view.WheelLayout>
+        </FrameLayout>
+        <FrameLayout
+            android:id="@+id/spin_spr_view"
+            android:layout_gravity="center"
+            android:layout_width="@dimen/spin_pointer_width"
+            android:layout_height="@dimen/spin_pointer_width">
+            <ImageView
+                android:id="@+id/spr_left"
+                android:layout_width="wrap_content"
+                android:src="@drawable/spin_wheel_spr_blue"
+                android:layout_gravity="left|center_vertical"
+                android:rotation="-30"
```

Comment:
You would not need to rotate the image immediately if it was not a rotated image in the first place. Why is the `spin_wheel_spr_blue` drawable rotated and not completely horizontal?